apps/kfp-tekton/upstream/base/installs/multi-user/pipelines-profile-controller/run_tests.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/sh
```

Comment:
everything under apps/*/upstream is not maintained by us and shall not be modified.

Reply:
I did the changes as requested